cinema/migrations/0003_alter_movie_image.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.1 on 2024-12-09 14:18
+
+import cinema.models
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("cinema", "0002_movie_image"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="movie",
+            name="image",
+            field=models.ImageField(
+                null=True, upload_to=cinema.models.image_custom_path
```

Comment:
Ensure that `cinema.models.image_custom_path` is defined in your models module. This function should handle the custom logic for generating the upload path for images.